site.yml

```diff
@@ -166,10 +183,10 @@
   roles:
     - role: openstack-database
       database_name: glance
-      tags: ['openstack', 'glance', 'control', 'ceph']
-
+      tags: ['openstack', 'glance', 'control']
     - role: glance
-      tags: ['openstack', 'glance', 'control', 'ceph']
```

Comment:
Because Glance can store images in ceph, if the cluster has ceph, and to do so, some ceph level things has to happen.